### PR TITLE
Feature/map interactive

### PR DESCRIPTION
**Pull request by Mozzaco**
La map est cliquable et renvoi la region en console.log. 
Les filtres seront ajouté plus tard.

### CODE REVIEW

**Comment by FPaurisse** [comment]
OK for the moment...

**Review by FPaurisse — APPROVED** [review]

File: src/components/Map.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import France from "@svg-maps/france.regions";
+import {SVGMap } from "react-svg-map"; 
+import "react-svg-map/lib/index.css";
+ 
+class Map extends React.Component {
+  constructor(props) {
+    super(props);
+  }
```
Comment (Aebrathia) [inline src/components/Map.js]: Your component is only presentational, it does not have state nor any logic. Use a function instead.

File: src/components/Map.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+ 
+  render() {
+    return <SVGMap map={France} onLocationClick={event => console.log(event.target.getAttribute('name'))} />;
+  }
+}
+ 
+export default Map;
```
Comment (Aebrathia) [inline src/components/Map.js]: Add missing newline

**Review by Aebrathia — CHANGES_REQUESTED** [review]
You have warnings in your console. Be careful not to push code with warnings or errors.

Your commit "region cliquable" is not clear enough and does not respect our conventions. It would have been better to write "Make regions clickable".

**Review by Aebrathia — APPROVED** [review]

**Comment by Aebrathia** [comment]
Congrats 2nd PR merged !

**Pull request merged**